Cython/Compiler/ExprNodes.py

```diff
@@ -8317,8 +8317,14 @@ def analyse_types(self, env, skip_children=False):
         return self
 
     def coerce_to_ctuple(self, dst_type, env):
+        if dst_type.is_reference:
```

Comment:
`PyrexTypes` has `remove_cv_ref`. It's probably overkill given how simple it is, but since it exists we may as well use it.